src/Identifier/LdapIdentifier.php

```diff
@@ -197,6 +212,29 @@ protected function _bindUser($username, $password)
         return null;
     }
 
+    /**
+     * Format LDAP entry data into associative array
+     *
+     * @param array $entry The entry
+     * @return array
+     */
+    protected function _formatEntry($entry)
```

Comment:
Plural form `_formatEntries()` would be better since it works on multiple entries.